examples/complete/main.tf

```diff
@@ -162,7 +162,7 @@ module "eks_node_group" {
 
   # Ensure ordering of resource creation to eliminate the race conditions when applying the Kubernetes Auth ConfigMap.
   # Do not create Node Group before the EKS cluster is created and the `aws-auth` Kubernetes ConfigMap is applied.
-  depends_on = [module.eks_cluster.kubernetes_config_map_id]
+  depends_on = [module.eks_cluster, module.eks_cluster.kubernetes_config_map_id]
```

Comment:
I would think `depends_on = [module.eks_cluster.kubernetes_config_map_id]` implies `depends_on = [module.eks_cluster]`, no?

Reply:
I would expect that you are right, but I did not want to spend the time to test it, and I ran into problems with the previous version.